beekeeper-integration-tests/src/test/java/com/expediagroup/beekeeper/integration/BeekeeperIntegrationTestBase.java

```diff
@@ -240,13 +255,14 @@ private HousekeepingPath createHousekeepingPath(String path, LifecycleEventType
         .build();
   }
 
-  private HousekeepingMetadata createHousekeepingMetadata(
+  protected HousekeepingMetadata createHousekeepingMetadata(
```

Comment:
Why did this change to protected?

Reply:
Because I use this method in `BeekeeperApiIntegrationTest` class too, where my integration tests for the API are

Reply:
This is an internal method. There are are other methods that allow you to insert metadata. Why can't you use them?